infra/cifuzz/config_utils.py

```diff
@@ -98,9 +98,15 @@ def platform(self):
 class RunFuzzersConfig(BaseConfig):
   """Class containing constant configuration for running fuzzers in CIFuzz."""
 
+  RUN_FUZZERS_MODES = {'batch', 'ci'}
+
   def __init__(self):
     super().__init__()
     self.fuzz_seconds = int(os.environ.get('FUZZ_SECONDS', 600))
+    self.run_fuzzers_mode = os.environ.get('RUN_FUZZERS_MODE', 'ci').lower()
+    if self.run_fuzzers_mode not in self.RUN_FUZZERS_MODES:
+      raise Exception('Invalid RUN_FUZZERS_MODE not one of allowed choices.',
+                      self.run_fuzzers_mode, self.RUN_FUZZERS_MODES)
```

Comment:
Will this Exception be hard to read, should place %s to indicate places of these two vars.

Reply:
Done.